From bcf259ecd5a458972ebf54b8c935ed9cf836bc29 Mon Sep 17 00:00:00 2001
From: wilkie <wilkie05@gmail.com>
Date: Thu, 19 Jan 2023 16:48:22 +0000
Subject: [PATCH] Offline updates.

1. unf_ext needs to be at least 0.0.7.4
2. docker compose needs to point to the db domain (not localhost)
---
 Gemfile                                      |  3 ++-
 config/development.yml.erb                   |  2 +-
 dashboard/config/environments/development.rb | 11 ++++++-----
 lib/cdo.rb                                   |  3 ++-
 4 files changed, 11 insertions(+), 8 deletions(-)

diff --git a/Gemfile b/Gemfile
index 0cbd62f80f2..cb6c78e16d2 100644
--- a/Gemfile
+++ b/Gemfile
@@ -277,7 +277,7 @@ gem 'rest-client', '~> 2.0.1'
 
 # A rest-client dependency
 # This is the latest version that's installing successfully
-gem 'unf_ext', '0.0.7.2'
+gem 'unf_ext', '0.0.7.4'
 
 # Generate SSL certificates.
 gem 'acmesmith', '~> 2.3.1'
@@ -357,3 +357,4 @@ gem 'pry'
 gem 'cld'
 
 gem 'crowdin-api', '~> 1.2.1'
+gem 'tzinfo-data'
diff --git a/config/development.yml.erb b/config/development.yml.erb
index fd074bba2fb..9f38aa8125b 100644
--- a/config/development.yml.erb
+++ b/config/development.yml.erb
@@ -13,7 +13,7 @@ db_writer_user: root
 db_writer_password: ''
 db_reader_user: root
 db_reader_password: ''
-db_writer: 'mysql://root@localhost/'
+db_writer: 'mysql://root:password@db/'
 
 firebase_name: cdo-v3-dev
 # provide a unique path for firebase channels data for development, to avoid conflicts in channel ids.
